components/gitpod-protocol/src/teams-projects-protocol.ts

```diff
@@ -45,6 +45,7 @@ export namespace Project {
 
     export interface Overview {
         branches: BranchDetails[];
+        prebuildState?: "suspended_on_inactivity" | "enabled";
```

Comment:
What is the reasoning behind making this an enum of potentially many "magic strings", as opposed to just a boolean flag `isConsideredInactive`?

My thinking is that we shouldn't over-engineer this unless there are very strong expectations that we'll add more states very soon / immediately.

Reply:
I tend to agree with Jan here.